Gemfile

```diff
@@ -38,6 +38,7 @@ gem 'rack-attack'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'm'
```

Comment:
heh, had a bit of a hard time searching for this 😂 - good add though 👍

Reply:
I am horrified that single-letter package names are ever accepted.